_data/toc.yaml

```diff
@@ -1324,9 +1304,26 @@ manuals:
       title: FAQs
     - path: /desktop/backup-and-restore/
       title: Back up and restore data
+    - sectiontitle: Previous versions
+      section:
+      - path: /desktop/previous-versions/3.x-windows/
+        title: Desktop for Windows 3.x
+      - path: /desktop/previous-versions/3.x-mac/
+        title: Desktop for Mac 3.x
+      - path: /desktop/previous-versions/2.x-windows/
+        title: Desktop for Windows 2.x
+      - path: /desktop/previous-versions/2.x-mac/
+        title: Desktop for Mac 2.x
+      - path: /desktop/previous-versions/edge-releases-windows/
+        title: Windows edge releases
+      - path: /desktop/previous-versions/edge-releases-mac/
+        title: Mac edge releases
+      - path: /desktop/previous-versions/archive-windows/
+        title: Older versions for Windows
+      - path: /desktop/previous-versions/archive-mac/
+        title: Older versions for Mac
     - path: /desktop/release-notes/
       title: Release notes
```

Comment:
Thanks for the PR @aevesdocker. Changes look really good. The only suggestion I have is to move the release notes above Previous versions.